src/graphs/conditional_independence_example.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+A graphical model to demonstrate conditional independence. Shows a graph that
+assumes travel cost is conditionally independent of travel time, conditional on
+travel distance.
+"""
+from causalgraphicalmodels import CausalGraphicalModel
+
+TRAVEL_TIME = 'Travel Time'
+TRAVEL_COST = 'Travel Cost'
+TRAVEL_DISTANCE = 'Travel Distance'
+
+
+EXAMPLE_GRAPH =\
+    CausalGraphicalModel(
+        nodes=[TRAVEL_TIME, TRAVEL_COST, TRAVEL_DISTANCE],
+        edges=[(TRAVEL_DISTANCE, TRAVEL_TIME), (TRAVEL_DISTANCE, TRAVEL_COST)]
```

Comment:
Is it worth it to take this out as well? and set up a drive alone nodes and drive alone edges variables outside of the graph variable?

Reply:
It could be, but I didn't know if it was really necessary as the example is only used in a single slide in the presentation, and we don't reuse the edges anywhere else (as far as I can see).